assets/js/modules/analytics/datastore/__fixtures__/account-ticket.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
So far all of the fixtures here are snapshots of the response for a given request. The name of the fixture should then match the request that returns it.

This seems to be the response for `fetchCreateAccount` so this should probably be called `create-account.json` (we don't have fixtures for the main action creator since it's `response` would be the same.

Reply:
Ah, I was naming it after the object it was a fixture for (the "Account Ticket". I renamed and changed it to `createAccount` in https://github.com/google/site-kit-wp/pull/1390/commits/20a4f07055283cb62d046f5534c6c2db474c230d.